docs/advanced/tracking.story.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Intro, Story } from '../../.storybook/components';
+
+<Meta title="Advanced/Event Tracking" />
+
+# Event Tracking
+
+<Intro>
+  Tracking user interactions enables you to learn about usage patterns and
+  improve the application. Circuit UI makes this straight-forward by integrating
+  <a href="https://github.com/sumup/collector">Collector</a>. Here are the steps
+  to enable and use it.
+</Intro>
+
+## Configuring tracking dispatch
+
+1. Wrap your application in root level with `TrackingRoot` and provide a dispatch function for your tracking service.
+2. Add `TrackingView` to a page or other meaningful view (e.g. a modal) in your application. (For more explanation about event schema, please refer to [Schema](https://github.com/sumup-oss/collector#schema) in Collector.)
+3. To ensure high data quality, the `label` tracking property is mandatory in Collector. Circuit UI components invoke the dispatch function only when a value for the `label` property is passed as part of the `tracking` prop.
+4. When a user interacts with a component, Circuit UI invokes the dispatch function with the standard event schema.
+
+```jsx
+import { TrackingRoot, TrackingView } from '@sumup/collector';
+
+export default function App() {
+  const sendTracking = payload => sendGA(payload);
```

Comment:
[in collector](https://github.com/sumup/collector#trackingroot) we recommend memoizing the dispatchHandler to avoid unnecessary renders. I think we should reflect it here